### PR TITLE
Adds Redis, WildFly, JBoss Data Virt, removes Memcached, updates docs

### PR DESCRIPTION
**Pull request by luciddreamz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,17 @@ Community Templates and ImageStreams are **not** provided or supported by Red Ha
 
 ## Contributing
 
+### Adding Your Template or ImageStream
+
+- Fork the https://github.com/openshift/library on github
+- Add your template or image stream to the **community.yaml** file in the top level of this project
+- Run the **import_content.py** script and make sure that your template(s) and/or image-stream(s) are processed and written to the correct directory under the **community** folder and that no errors have occurred.
+- Commit your changes to your fork of the github repo
```
Comment (bparees) [inline README.md]: someone didn't follow their own instructions.

please include the result of the import so i can actually see what's getting pulled in/changed as a result of this.

File: community.yaml [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-  php:
+  laravel:
     templates:
       - location: https://raw.githubusercontent.com/luciddreamz/laravel-ex/master/openshift/templates/laravel-mysql.json
         docs: https://github.com/luciddreamz/laravel-ex/blob/master/readme.md
       - location: https://raw.githubusercontent.com/luciddreamz/laravel-ex/master/openshift/templates/laravel-postgresql.json
         docs: https://github.com/luciddreamz/laravel-ex/blob/master/readme.md
       - location: https://raw.githubusercontent.com/luciddreamz/laravel-ex/master/openshift/templates/laravel-sqlite.json
         docs: https://github.com/luciddreamz/laravel-ex/blob/master/readme.md
-  memcached:
```
Comment (bparees) [inline community.yaml]: 1) why are we removing it?

2) more a question for @coreydaley but does the import script clean up files/directories that are deleted from the yaml manifests?

Comment (coreydaley) [inline community.yaml]: @bparees Yes, it deletes the whole official and community directories and rebuilds them: https://github.com/openshift/library/blob/master/import_content.py#L255

Comment (luciddreamz) [inline community.yaml]: I added Memcached as an example of a community image stream with the image sourced from Docker Hub. The image stream / example never actually worked. This was part of my pre-Library that got carried over here. WildFly provides a better, actual working example.

Comment (bparees) [inline community.yaml]: > The image stream / example never actually worked. 

ouch.

ok.

File: README.md (hunk 1 above)
Comment (luciddreamz) [inline README.md]: Awww. I didn't write that part of the instructions and I still don't have this stuff installed locally =\

**Comment by luciddreamz** [comment]
Waiting on: https://github.com/jboss-fuse/application-templates/pull/22

**Comment by luciddreamz** [comment]
@bparees @coreydaley PR updated

**Comment by bparees** [comment]
lgtm.

**Pull request merged**